benchmarks/test_peer_selection.py

```diff
@@ -39,6 +39,7 @@ class FakeConnection(object):
     def __init__(self, hostport):
         self.hostport = hostport
         self.closed = False
+        self.direction = object()
```

Comment:
It is because in the "self.score_calculator.get_score(peer)" will need to get the incoming connections which requires the direction attributes. So I added it here.